CONTRIBUTING.md

```diff
@@ -0,0 +1,10 @@
+# Contributing
+
+Thanks for being willing to contribute!
+
+## Pull Request Process
+
+1. Ensure any install or build dependencies are removed before the end of the layer when doing a build.
+2. Use the PR template to provide important information about your work.
+3. Your pull request will be reviewed in two separate stages. The first stage requires that an authorized contributor 
+signs off on the changes. The second stage required the head reviewer to take a look and make the appropriate changes.
```

Comment:
quik question but do we want to add tests as a step for the PR process?

Reply:
+1 @maxnawa31